configure.ac

```diff
@@ -6,7 +6,7 @@
 # Use and distribution licensed under the BSD license.  See
 # the COPYING file in this directory for full text.
 
-AC_INIT([libdrizzle],[5.1.4],[http://bugs.launchpad.net/drizzle],[libdrizzle],[http://drizzle.org/])
```

Comment:
> I will look into that.

Trying to define **version** dynamically in `AC_INIT` without success. 
Any idea how to do that?